src/codegen.cpp

```diff
@@ -1080,6 +1108,9 @@ jl_code_instance_t *jl_compile_linfo(jl_method_instance_t *mi, jl_code_info_t *s
 
     // Codegen lock held in this block
     {
+        // Instrument the time spent in this block for this method instance
+        CompileTiming _c(mi);
```

Comment:
This will not time LLVM optimizations; see the comment at the top of this function.

Reply:
Ah, yeah, we just realized that as well.

I've pushed up new commits that move the timings to `jitlayers.cpp`. Now the trouble is trying to figure out _which_ method we're compiling! 😅

Reply:
Sorry i hadn't pushed that until just now